Modules/IO/JPEG/src/itkJPEGImageIO.cxx

```diff
@@ -321,6 +319,13 @@ JPEGImageIO::ReadImageInformation()
   // ability to scale an image on input).
   jpeg_calc_output_dimensions(&cinfo);
```

Comment:
Maybe call this _jpeg_calc_output_dimensions(&cinfo)_ in _Read_ function too, from the comment it looks like there may be situation where _cinfo.image_height_ etc. may be affected? And m_Dimensions are saved here from _cinfo_. Not 100% sure. Gimp code doesn't have that _jpeg_calc_output_dimensions_ call.